util/get_holes.py

```diff
@@ -0,0 +1,58 @@
+import json
+from pathlib import Path
+import requests
+from time import sleep
```

Comment:
I think this import is unused.

Reply:
You're right, I took those out. I'll remove it in the next commit.